central/apitoken/datastore/datastore.go

```diff
@@ -35,3 +37,8 @@ func New(storage store.Store) DataStore {
 func NewPostgres(pool postgres.DB) DataStore {
 	return newPostgres(pool)
 }
+
+// NewTestPostgres provides a datastore connected to postgres for testing purposes.
```

Comment:
Couldn't this function be defined in `datastore_test.go`?

Reply:
No I don't think so, golang does not export from `_test.go` and we use this in the pruning tests.